test/components/admin/editions/show/sidebar_actions_component_test.rb

```diff
@@ -0,0 +1,197 @@
+# frozen_string_literal: true
```

Comment:
Is this intended?

Reply:
It's a ruby "magic comment" that's present in all of the ViewComponent files to improve memory performance by only defining a single instance of each unique string.